src/mmcmds.c

```diff
@@ -4674,8 +4674,13 @@ void eraseSource(void)    /* ERASE command */
 } /* eraseSource */
 
 
-/* If verify = 0, parse the proofs only for gross error checking.
-   If verify = 1, do the full verification. */
+/* If verifyFlag = VERIFY_NONE, parse the proofs only for gross error checking.
+   If verifyFlag includes VERIFY_FULL, do the full verification.
```

Comment:
It seems this flag only controls a progress bar and a couple of result print outs. Please be more specific.